api/pkg/apis/v1alpha1/providers/config/catalog/catalogprovider.go

```diff
@@ -199,7 +215,7 @@ func (m *CatalogConfigProvider) traceValue(v interface{}, localcontext interface
 	}
 }
 func (m *CatalogConfigProvider) Set(object string, field string, value interface{}) error {
-	catalog, err := utils.GetCatalog(context.TODO(), m.Config.BaseUrl, object, m.Config.User, m.Config.Password)
+	catalog, err := utils.GetCatalog(context.TODO(), m.Config.BaseUrl, object, m.Config.User, m.Config.Password, "")
```

Comment:
So this (and methods below) still access just the default namespace?

Reply:
Yes. Shall we update it in this PR? I will look into that.